src/main/kotlin/app/revanced/patches/urbandroid/sleep/annotations/UnlockPremiumCompatibility.kt

```diff
@@ -0,0 +1,9 @@
+package app.revanced.patches.urbandroid.sleep.annotations
```

Comment:
```suggestion
package app.revanced.patches.sleepasandroid.misc.annotations
```
I think it would be better to rename package to the app name and the category

Reply:
yeah, makes sense